metaflow/plugins/argo/argo_workflows.py

```diff
@@ -771,7 +771,7 @@ def _container_templates(self):
                         "METAFLOW_RETRY_COUNT": retry_count,
                         "METAFLOW_PRODUCTION_TOKEN": self.production_token,
                         "ARGO_WORKFLOW_TEMPLATE": self.name,
-                        "ARGO_WORKFLOW_NAME": "{{workflow.name}}",
+                        "ARGO_WORKFLOW_NAME_METAFLOW": "{{workflow.name}}",
```

Comment:
This is a temporary renaming that should be reverted once `ARGO_WORKFLOW_NAME` is removed from pod defaults, blocked by on-going upgrades that I'm working on right now.

Reply:
Can you please add a TODO and JIRA task to undo this?